src/Slider.jsx

```diff
@@ -727,7 +770,18 @@ class Rheostat extends React.Component {
               role="slider"
               style={handleStyle}
               tabIndex={0}
-            />
+            >
+              {
+                handleTooltip && (
```

Comment:
nit: this should be `{handleTooltip && (` and everything else should be summarily de-indented